businesscentral/modules/businesscentral/module.yaml

```diff
@@ -25,8 +25,8 @@ ports:
 - value: 8001
 artifacts:
 - name: "rhpam_business_central_distribution.zip"
-  # rhpam-7.12.0-business-central-eap7-deployable.zip
-  md5: "97367258e9ecc3f1b65c72a2648f5ffc"
+  # rhpam-7.12.1-business-central-eap7-deployable.zip
+  md5: "f03730f846e875ddbbd6ff43b1e3b480"
```

Comment:
are `RHPAM_BUSINESS_CENTRAL_VERSION` and `PRODUCT_VERSION` ok? shouldn't they be `7.12.1`?